src/features/profile/Profile.tsx

```diff
@@ -87,26 +86,20 @@ const profile: React.FC<Props> = ({
           </Text>
           <RadioButton
             isChecked={theme === Themes.dark}
-            onPress={() => {
-              setTheme(Themes.dark);
-            }}
+            onPress={() => setTheme(Themes.dark)}
```

Comment:
Not sure, its better to use useCallback and pass here as a call back not create inside a new arrow func :)